python/cudf/cudf/_lib/cpp/wrappers/decimals.pxd

```diff
@@ -1,12 +1,17 @@
-# Copyright (c) 2021, NVIDIA CORPORATION.
+# Copyright (c) 2021-2022, NVIDIA CORPORATION.
+
 from libc.stdint cimport int32_t, int64_t
 
+from cudf._lib.cpp.types cimport int128
+
 
 cdef extern from "cudf/fixed_point/fixed_point.hpp" namespace "numeric" nogil:
     # cython type stub to help resolve to numeric::decimal64
     ctypedef int64_t decimal64
     # cython type stub to help resolve to numeric::decimal32
     ctypedef int64_t decimal32
+    # cython type stub to help resolve to numeric::decimal128
+    ctypedef int128 decimal128
```

Comment:
I think you can just use a C-name specification here to trick Cython into generating the right code:

```
ctypedef int64_t decimal128 "__int128_t"
```

Reply:
Synced offline with @shwina and removed the cpp changes with a similar cython hack.